src/Dev/TestKernel.php

```diff
@@ -48,15 +48,16 @@ protected function getIncludeTests()
 
 
     /**
-     * @param string[] $ciLibs
+     * Set a list of CI configurations that should cause a module's test not to be added to a manifest
+     * @param string[] $ciConfigs
      */
-    public function setIgnoreCILibraries(array $ciLibs): self
+    public function setIgnoreCIConfigs(array $ciLibs): self
```

Comment:
```suggestion
    public function setIgnoreCIConfigs(array $ciConfigs): self
```

Reply:
Should we also use an adjective everywhere, i.e. `setIgnored...`? There seems to be a mix at the moment.

Reply:
Done.